specutils/io/default_loaders/sdss_v.py

```diff
@@ -462,14 +467,17 @@ def load_sdss_mwm_1d(file_obj, hdu: Optional[int] = None, **kwargs):
             for i in range(len(hdulist)):
                 if hdulist[i].header.get("DATASUM") != "0":
                     hdu = i
+                    print('HDU not specified. Loading spectrum at (HDU{})'.
```

Comment:
Is this just a warning or is specifying an HDU required?   For production, we use Jdaviz to load the file, and want by default all data extensions loaded, without having to specify an HDU, which I think will the most common use case by users as well.  Does this result in many constant warning messages printed to the user?

Reply:
I'm not sure how we want to handle this case generally. `mwm` files contain a max of 4 extensions which can each have N spectra. These shouldn't really be loaded into a single `Spectrum1D` object because APOGEE/BOSS are at different wavelength ranges.

Reply:
Yeah I agree that the APOGEE/BOSS extensions should not be loaded into a single Spectrum1D.  They should be loaded into a SpectrumList.  I think the warning is fine.  It's useful for anyone using `specutils` directly, and will be buried within Specviz and Zora.